SIT/environment.sh

```diff
@@ -69,7 +69,8 @@ fi
 # These variables are hard-coded for now
 SUBNET=$(echo ${CIDR} | cut -d. -f1-3)
 CASS_VERSION=4
-CDM_VERSION=latest
+#CDM_VERSION=latest
+CDM_VERSION=ms-jdk17
```

Comment:
This is **a temporary commit** for all the GHA jobs on this PR to work correctly. We will revert back with the `latest` image post performing the following:

- [ ] Merge this PR into `main` branch
- [ ] Publish a new Docker image using the updated codebase